source/en/api/api_bandit.rst

```diff
@@ -55,6 +57,19 @@ We show each field below:
 
         * range of value: 0 <= ``seed`` <= :math:`2^{32} - 1`
 
+   epsilon_decreasing
+    :decreasing_rate:
+        Decreasing parameter of epsilon. The bigger this parameter becomes, the more slowly epsilon dcreases. (Float)
```

Comment:
typo: dcreases

Reply:
As commented in Japanese docs, I think `epsilon` is strange.
How about like below?
"Decreasing rate for the probability of selecting arms randomly. The bigger this parameter is, the more slowly the probability decreases."